build.sh

```diff
@@ -261,6 +263,10 @@ for i in "$@"
         arm64)
         __BuildArch=arm64
         __MSBuildBuildArch=arm64
+        
+        # if we are on arm64 then we need clang 3.8
+
+        __ClangMinorVersion=8
```

Comment:
+1 

clang 3.8 is moving target because of it was not released yet. It is not a good idea to require it unless it is absolutely needed.